test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -51,6 +51,17 @@ get_stat(scoped_cursor &cursor, int stat_field, int64_t *valuep)
     testutil_check(cursor->get_value(cursor.get(), &desc, &pvalue, valuep));
 }
 
+/* Consider generating this programmatically in `stat.py`. */
+inline int
+get_stat_field(const std::string &name)
+{
+    if (name == "cache_hs_insert")
+        return WT_STAT_CONN_CACHE_HS_INSERT;
+    else if (name == "cc_pages_removed")
+        return WT_STAT_CONN_CC_PAGES_REMOVED;
+    testutil_die(-1, "get_stat_field: Stat \"%s\" is unrecognized", name.c_str());
```

Comment:
Should we use EINVAL instead of -1 ? I think this is what we have been doing.

Reply:
fe68600